card.schema.json

```diff
@@ -15,6 +15,17 @@
                     "enum": ["X", "-"]
                 }
             ]
+        },
+        "erratum": {
+            "type": "object",
+            "properties": {
+                "issuer": { "enum": ["Fantasy Flight Games", "The Conclave"] },
+                "version": { "type": "string", "minLength": 1 },
+                "link": { "type": "string", "format": "uri" },
+                "text": { "type": "string", "minLength": 1 },
+                "notes":  { "type": [ "string", "null" ] }
```

Comment:
Notes field seems like a good compromise until we need to deal with stat changes (and may be good enough there)!